src/main/java/com/squareup/javawriter/ParameterWriter.java

```diff
@@ -18,15 +18,24 @@
 import java.io.IOException;
 
 public final class ParameterWriter extends VariableWriter {
+  private boolean isVararg;
+
   ParameterWriter(TypeName type, String name) {
     super(type, name);
   }
 
+  public void setIsVararg(boolean isVararg) {
```

Comment:
Odd for a setter to not take a value, but I find the `boolean` to be useless as well.